examples/applications/plot_model_complexity_influence.py

```diff
@@ -162,8 +191,43 @@ def _count_nonzero_coefficients(estimator):
      'prediction_performance_label': 'MSE',
      'n_samples': 30},
 ]
+
+
+##############################################################################
+# Run the code and plot the results
+# ---------------------------------
+# We are ready to run all our functions by looping though the configurations we
+# set previously and then plotting the influence of varying parameters on the
+# complexity and the latency of each model.
+#
+
+
+def plot_influence(conf, mse_values, prediction_times, complexities):
+    """
+    Plot influence of model complexity on both accuracy and latency.
+    """
+    plt.figure(figsize=(12, 6))
+    host = host_subplot(111, axes_class=Axes)
+    plt.subplots_adjust(right=0.75)
+    par1 = host.twinx()
+    host.set_xlabel('Model Complexity (%s)' % conf['complexity_label'])
+    y1_label = conf['prediction_performance_label']
+    y2_label = "Time (s)"
+    host.set_ylabel(y1_label)
+    par1.set_ylabel(y2_label)
+    p1, = host.plot(complexities, mse_values, 'b-', label="prediction error")
+    p2, = par1.plot(complexities, prediction_times, 'r-',
+                    label="latency")
+    host.legend(loc='upper right')
+    host.axis["left"].label.set_color(p1.get_color())
+    par1.axis["right"].label.set_color(p2.get_color())
+    plt.title("Influence of varying '%s' on %s" % (conf['changing_param'],
+                                                   conf['estimator'].__name__))
+
+
 for conf in configurations:
     prediction_performances, prediction_times, complexities = \
         benchmark_influence(conf)
     plot_influence(conf, prediction_performances, prediction_times,
                    complexities)
+plt.show()
```

Comment:
I think that we are missing the most important message :)
We need to add a description and an analysis of the results (we probably did not have this before but it would be a great add-on).

```
First, we can analyze the plots obtained from the benchmark. Relaxing the `L1` penalty
in the SGD classifier reduces the prediction error but leads to an increase in the training
time.

We can draw a similar analysis regarding the training time which increases with the
number of support vectors with a Nu-SVR. However, we observed that there is an optimal
number of support vectors which reduces the prediction error. Indeed, too few support
vectors lead to an under-fitted model while too many support vectors lead to an over-fitted
model.

The exact same conclusion can be drawn for the gradient-boosting model. The only
the difference with the Nu-SVR is that having too many trees in the ensemble is not as
detrimental.

As a conclusion, we can deduce the following insights:

* a model which is more complex (or expressive) will require a larger training time;
* a more complex model does not guarantee to reduce the prediction error.

These aspects are related to model generalization and avoiding model under-fitting or
over-fitting.
```